diff --git a/node_modules/@wessberg/rollup-plugin-ts/dist/cjs/index.js b/node_modules/@wessberg/rollup-plugin-ts/dist/cjs/index.js
index 205e06f..7502305 100644
--- a/node_modules/@wessberg/rollup-plugin-ts/dist/cjs/index.js
+++ b/node_modules/@wessberg/rollup-plugin-ts/dist/cjs/index.js
@@ -1210,7 +1210,7 @@ function getAliasedDeclaration(node, typeChecker) {
     try {
         symbol = node == null ? undefined : "valueDeclaration" in node || "declarations" in node ? node : typeChecker.getSymbolAtLocation(node);
     }
-    catch {
+    catch (e) {
         // Typescript couldn't produce a symbol for the Node
     }
     if (symbol == null)
@@ -1228,7 +1228,7 @@ function getAliasedDeclaration(node, typeChecker) {
                         : undefined;
         }
     }
-    catch { }
+    catch (e) { }
     return valueDeclaration;
 }
 
